### ISSUE TITLE
Can not retrain after updating the Repo

### ISSUE THREAD

**Issue by beichao1314** [OpenNMT/OpenNMT#365]
Hello, developers.
After updating OpenNMT, I retrain the model that trained from the version about `Enable Adam learning rate decay and restarts`. But I got the error:
```
/home/beichao/tools/torch/install/bin/luajit: ./onmt/modules/Decoder.lua:117: attempt to perform arithmetic on field 'base_scheduled_sampling' (a nil value)
stack traceback:
	./onmt/modules/Decoder.lua:117: in function 'scheduledSamplingDecay'
	./onmt/Seq2Seq.lua:188: in function 'updateRates'
	./onmt/train/Trainer.lua:191: in function 'closure'
	./onmt/utils/Parallel.lua:79: in function 'launch'
	./onmt/train/Trainer.lua:185: in function 'trainEpoch'
	./onmt/train/Trainer.lua:428: in function 'train'
	train.lua:173: in function 'main'
	train.lua:178: in main chunk
	[C]: in function 'dofile'
	...ools/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00406670
```
And my config is as fellow：
```
data = data/enzh_3k5-train.t7
layers = 2
rnn_size = 1024
encoder_type = gnmt
save_every = 30000
report_every = 100
max_batch_size = 64
save_model = model/enzh_3k5
gpuid = 1
validation_metric = bleu
optim = adam
learning_rate = 0.0002
bridge = dense_nonlinear
global_attention = dot
use_pos_emb = 1
dropout_type = variational
train_from = model/enzh_3k5_checkpoint.t7
continue = 1
```
And then, it works well when I retrain on the old version.

**Comment by jsenellart** [comment]
normally fixed with 2fec89b. Can you confirm?

**Comment by beichao1314** [comment]
It works well, thanks for all your work. :D
